packages/compiler/test/projection/projector-identity.test.ts

```diff
@@ -126,6 +126,22 @@ describe("compiler: projector: Identity", () => {
       strictEqual(projectResult.type.properties.get("name")?.model, projectResult.type);
     });
 
+    it("link projected model with sourceModels", async () => {
+      const code = `
+        @test("target") model Foo is Bar {
+          ...Spreadable
+        }
+        model Bar {}
+        model Spreadable {}
+      `;
+      const projectResult = await projectWithNoChange(code, "Model");
+      const [sourceBar, sourceSpreadable] = projectResult.type.sourceModels.map((x) => x.model);
+      const Spreadable = projectResult.globalNamespace.models.get("Spreadable")!;
+      const Bar = projectResult.globalNamespace.models.get("Bar")!;
+      strictEqual(sourceBar, Bar);
```

Comment:
Nit: Really minor, but should we have a truthy check for `sourceBar` and `sourceSpreadable`? I'm imagining the worst case where everything we're checking happens to be `undefined` and so matches.